src/pages/NewListing.jsx

```diff
@@ -419,17 +433,45 @@ const NewListing = () => {
           }
         </FlexRow>
         <FlexRow>
-        {<CheckCircle color="success" />}
-          <div className="zoom-burn-fee">
-            Zoom <StyledLogo src={zoomLogo} /> Burn Fee:{' '}
-            {data && data.zoomBurnFee
-              ? data.zoomBurnFee * numberOfSelectedCards
-              : 0}{' '}
-            {' '}
-          </div>
+          <FlexColumn>
+            <FlexRow>
+              {
+                !exceedZoomAllowance &&
+                (
+                  <CheckCircle color="success" />
+                )
+              }
+              <div className="zoom-burn-fee">
+                Zoom <StyledLogo src={zoomLogo} /> Burn Fee:{' '}
```

Comment:
nit, can you remove the spaces and use margin and span to create spacing between texts?

Reply:
Yep. I will do that.